backends/dropbox/index.js

```diff
@@ -7,6 +7,9 @@ export default class Dropbox extends OAuthBackend {
 	static apiDomain = "https://api.dropboxapi.com/2/";
 	static oAuth = "https://www.dropbox.com/oauth2/authorize";
 	static fileBased = true;
+	static urls = [
+		{"hostname": "{*.}?dropbox.com"}
```

Comment:
Do I understand correctly that we don't need it to be the valid JSON object here? Is it just a copy-paste from the playground? :)

Should we go with
```js
{hostname: "{*.}?dropbox.com"}
```